float32 PanCmdPos
float32 TiltCmdPos